scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -539,6 +544,33 @@ public static void PublishSharedFramework(BuildTargetContext c, string outputDir
                 var runtimeGraphGeneratorProject = Path.Combine(Dirs.RepoRoot, "tools", runtimeGraphGeneratorName);
                 var runtimeGraphGeneratorOutput = Path.Combine(Dirs.Output, "tools", runtimeGraphGeneratorName);
 
+                // Override the NuGet version the RuntimeGraphGenerator depends on.
+                // NOTE: This is necessary for bootstrapping https://github.com/dotnet/cli/issues/2874.
+                //       This code will be removed after change has made its way into stage 0.
+                var originals = new[]
```

Comment:
It's fine for this PR, but do consider creating a new Type for this kind of bootstrapping stuff e.g. NuGetLowerCaseBootstrapping and putting all the temporary logic in there. We're at a bit of risk, albeit controlled, that these inlined changes will get refactored or further changed before we can back them out. A separate type would also just own the responsibility of bootstrapping so folks could look at it all in one place and consider bringing it back when we need to do something like this again.